spec/installer-spec.js

```diff
@@ -26,6 +26,10 @@ test('creates a nuget package and installer', async t => {
     iconUrl: 'https://boo'
   };
 
+  if (process.platform === 'win32') {
+    options.msi = true;
```

Comment:
Without this a MSI won't be created and therefore the unit test would fail on Windows.